### PR TITLE
add validator option --accounts-db-skip-shrink

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
When restarting validators with large snapshots, shrink can take hours. This option is useful to more quickly get to other troublesome parts of validator startup while debugging the system.
#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by jeffwashington** [comment]
verified this disables shrink on a validator

File: validator/src/main.rs [hunk 1]
```diff
@@ -1812,6 +1812,12 @@ pub fn main() {
                 .long("no-accounts-db-caching")
                 .help("Disables accounts caching"),
         )
+        .arg(
+            Arg::with_name("accounts_db_skip_shrink")
+                .long("accounts-db-skip-shrink")
+                .help("Enables faster starting of validators by skipping shrink. \
+                      This option if for use during testing."),
```
Comment (CriesofCarrots) [inline validator/src/main.rs:1819]:
```suggestion
                      This option is for use during testing."),
```

**Review by CriesofCarrots — DISMISSED** [review]
Lgtm! one typo nit
I'm thinking of plumbing this into the `multinode/net` scripts. Are there any other accounts db or index validator parameters that you think would be useful to be able to toggle from the testnet run commands?

**Comment by jeffwashington** [comment]
> Any plans to backport this one? If not, can we add a hidden stub flag on v1.7 for compatibility

I have no objections to back porting it. It should not add risk if I did it right ;-)

**Pull request merged**